pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -103,6 +105,33 @@ func (z *zedrouter) getNIPortConfig(
 		if port == nil {
 			continue
 		}
+		ntpServerIPs, ntpServerDomainsOrIPs := types.GetNTPServers(*z.deviceNetworkStatus, port.IfName)
+		ntpServers := make([]net.IP, 0, len(ntpServerDomainsOrIPs))
+		for _, ntpServer := range ntpServerDomainsOrIPs {
+			ip := net.ParseIP(ntpServer)
+			if ip != nil {
+				ntpServers = append(ntpServers, ip)
+				continue
+			}
+			dnsResponses, err := devicenetwork.ResolveWithPortsLambda(
+				ntpServer,
+				*z.deviceNetworkStatus,
+				devicenetwork.ResolveCacheWrap(func(domain string, dnsServerIP, srcIP net.IP) ([]devicenetwork.DNSResponse, error) {
```

Comment:
I think that you can just pass `devicenetwork.ResolveCacheWrap` as the argument, no need to wrap it.